Cargo.toml

```diff
@@ -10,14 +10,14 @@ keywords = ["serde", "pyo3", "python", "ffi"]
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-pyo3 = ">=0.23.0, <0.25.0"
-serde = "1.0.190"
+pyo3 = ">=0.23.0, <0.26.0"
+serde = "1"
```

Comment:
Since `serde = "1.0.190"` means `1.0.190 <= version < 2.0.0` (not `< 1.1`), this change only allows `1.0.189` or older. There is no reason to use old patch version. This change is not acceptable.